dev-resources/less/pages/ilmoitukset.less

```diff
@@ -21,6 +21,82 @@
   .boolean > .checkbox {
     padding-top: 0;
   }
+
+  .urakka {
+    width: 140px;
+
+    @media (max-width: 1000px) {
+      width: 120px;
+    }
+  }
+
+  .saapunut {
+    width: 125px;
+
+    @media (max-width: 1150px) {
+      width: 96px;
+    }
+  }
+
+  .tyyppi {
+    width: 70px;
+  }
+
+  .selite {
+    width: 200px;
+
+    @media (max-width: 1200px) {
+      width: 150px;
+    }
+
+    @media (max-width: 1050px) {
+      width: 100px;
+    }
+  }
+
+  .lisatieto {
+    @media (max-width:  900px) {
```

Comment:
Jäin miettimään: meillä on variabless.less:ssä määritelty tietyt kynnysarvot leveyteen, milloin käli alkaa reagoida, esim 480px, 768px, 992px. Näille on myös LESS-muuttujat.
Nyt nämä uudet uhkaa jäädä vähän taikanumeroiksi - voisiko niistä tehdä edes LESS-muuttujat jotta jatkossa voisi jokin muu näkymä näitä hyödyntää? Toisaalta: Olisiko mahdollista käyttää niitä @screen-sm  jne valmiita muuttujia, mietin voiko ne tuoda etuja siinä että ruutu "hyppisi" mahdollisimman harvoin ja silloinkin synkassa Bootstrapin scaffoldingin kanssa (esim grid.less ja bootstrap_overrides.less sisältää näiden käyttöjä)

Reply:
Siirretty muuttujaan, jossa selitetty mistä nämä breakpointit tulevat.